### PR TITLE
CACTUS-1060 :: Remove `vh` styles from Dimmer

### PR DESCRIPTION
**Pull request by daniloPenaR**
[CACTUS-1060](https://repayonline.atlassian.net/browse/CACTUS-1060)

I guess the title of this should be different. 

I spent more time struggling with how to build a local version of the `react-ui-base-image` to test this in channels. But I got it and it worked. 

Feel free to test by yourself. I did it with iOS and android.

### CODE REVIEW

**Comment by Dhalton** [comment]
This doesn't fix the original issue as you can see in this screenshot:

Screen Shot 2022-10-25 at 10 05 15 AM

To really test this, you have to undo my short-term fix in Channels, which is [here](https://github.com/repaygithub/repay-api/blob/master/ebpp_ui/app/components/SchedulerModalV2/styleOverrides.js#L25-L27)

File: modules/cactus-web/src/Dimmer/Dimmer.tsx [hunk 1]
```diff
@@ -27,6 +27,7 @@ export const Dimmer = styledUnpoly(DimmerBase)`
   left: 0;
   width: 100vw;
   height: 100vh;
+  padding-bottom: calc(100vh-100%);
```
Comment (maxrcollins) [inline modules/cactus-web/src/Dimmer/Dimmer.tsx:30]: oof. Make sure you test this in a bunch of browsers. Calc with values like this could be huge pain.

**Comment by wilysword** [comment]
Since it apparently didn't work, I guess just go back to the original ticket description of changing `height: 100vh` to `height: 100%`, see if that works.

**Review by Dhalton — APPROVED** [review]
Tested on several device/browser combos including iOS Safari/Chrome, Android Chrome/Firefox, and IE11. LGTM

**Review by maxrcollins — APPROVED** [review]
Tested Chrome, Phone Chrome, IOS, Firefox. Looks good to me.

**Pull request merged**